src/shared/test/integration/rpc-bridge-test.js

```diff
@@ -0,0 +1,206 @@
+import Bridge from '../../bridge';
+
+describe('RPC-Bridge integration', () => {
+  const sandbox = sinon.createSandbox();
+  let clock;
+  let port1;
+  let port2;
+  let bridges = [];
+
+  function createBridge() {
+    const bridge = new Bridge();
+    bridges.push(bridge);
+    return bridge;
+  }
+
+  function waitForMessageDelivery() {
+    return new Promise(resolve => setTimeout(resolve, 0));
+  }
+
+  beforeEach(() => {
+    clock = sinon.useFakeTimers();
+    const channel = new MessageChannel();
+    port1 = channel.port1;
+    port2 = channel.port2;
+  });
+
+  afterEach(() => {
+    bridges.forEach(bridge => bridge.destroy());
+    sandbox.restore();
+    clock.restore();
+  });
+
+  context('establishing a connection', () => {
```

Comment:
`describe` is more appropriate than `context` here. Functionally I believe they are exactly the same. However `context` is normally used to refer to a situation or circumstance (eg. "browser doesn't support feature X") whereas `describe` refers to _what_ is being tested, whether that is a specific class/function/method, as in most unit tests, or a process, as is the case here.